components/profile-controller/README.md

```diff
@@ -0,0 +1,72 @@
+# Profile CRD
+
+Kubeflow Profile CRD is designed to solve access management within multi-user kubernetes cluster.
+
+Profile access management provide namespace level isolation based on:
+
+- k8s rbac access control
+- Istio rbac access control
+
+**Resources managed by profile CRD:**
+
+Each profile CRD will manage one namespace (with same name as profile CRD), and will have one owner.
+Specifically, each profile CRD will manage following resources:
+
+- Namespace
+- K8s rbac Role & Rolebinding to grant profile owner access to above namespace via k8s API (kubectl)
+- Istio namespace-scoped gateway & rbac permission to grant profile owner access to above namespace via Istio (browser)
+- Setup namespace-scoped service-accounts `editor` and `viewer` to be used by user-created pods in 
+- resource quota management (coming)
+
+## Supported platforms and prerequisites:
+
+**GCP**
+- All users should have IAM permission `Kubernetes Engine Cluster Viewer`
+  - This is needed in order to get cluster access by `gcloud container clusters get-credentials`
+- kubeflow cluster with version v0.6
+
+## Manage access control and resources
+
+### manual access management by admin
+
+Cluster admin will manage access management for cluster users:
+
+**To create and reserve namespace `ns1` for user `abc@def.com`**
+- create profile:
+```
+apiVersion: kubeflow.org/v1alpha1
+kind: Profile
+metadata:
+  name: ns1
+spec:
+  owner:
+    kind: User
+    name: abc@def.com
+```
+
+**To revoke access to namespace `ns1` from user `abc@def.com` and delete namespace `ns1`**
+- delete profile ns1:
+```
+kubectl delete profile ns1
+```
+
+You should have kubeflow components deployed inside your k8s cluster. Generated ksonnet application is store in [app-dir](./bootstrapper.yaml#L65).
```

Comment:
Are these instructions for the admin?

- Should probably be updated for kustomize.
- Would each user have a ConfigMap that would define what components they would like to launch in their profile namespace? 
- Would the ConfigMap be created by them within their profile namespace?
- Have we figured out how this works - I thought Jobs, Notebook Pods would be launched into the profile namespace from the controller running in the kubeflow namespace?
- May want to point to a CUJ or similar
-

Reply:
Oh, this seems copied from bootstrap part, removed.